snowplow-tracker/src/main/java/com/snowplowanalytics/core/tracker/TrackerEvent.kt

```diff
@@ -120,4 +133,18 @@ class TrackerEvent @JvmOverloads constructor(event: Event, state: TrackerStateSn
             Parameters.UNSTRUCTURED
         )
     }
+    
+    private fun wrapWebViewToPayload(toPayload: Payload, base64Encoded: Boolean) {
```

Comment:
Very small nit: This is kind of a destructive function – if you call it once, it works fine, but if you call it twice, it will lose the self-describing data. It'd be better if it didn't call `remove` on `payload` and instead of filtered the properties of the `payload` when adding them over to `toPayload` like this:

```kt
        toPayload.addMap(payload.filterNot { it.key == Parameters.WEBVIEW_EVENT_DATA })
```